Pan and zoom as workflow config and make public

srallen opened the pull request:
Makes pan and zoom a workflow config and public. Should not be merged until zooniverse/Panoptes#1991 is resolved.
# Review Checklist
- [x] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [x] Does it work on mobile?
- [x] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [x] Did you deploy a staging branch? https://make-pan-zoom-public.pfe-preview.zooniverse.org/
## Optional
- [ ] If it's a new component, is it in ES6? Is it clear of warnings from ESLint?
- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [x] If changes are made to the classifier, does the dev classifier still work?

srallen commented:
This is now ready for review since zooniverse/Panoptes#1991 is done. @simensta are you available to review? @mrniaboc do you want to coordinate a time to announce this when we merge this PR?

simensta commented on app/pages/lab/workflow.cjsx:
```diff
@@ -304,6 +304,22 @@ EditWorkflowPage = React.createClass
+            <div>
+              <AutoSave resource={@props.workflow}>
+                <span className="form-label">Pan and zoom</span><br />
+                <small className="form-help">Pan and zoom allows the user to zoom in and out and pan the subject in the subject viewer.</small>
```
Do we need to indicate that Pan and Zoom will only work for images?

mschwamb replied:
We use the term classification interface in the glossary that @BNewing is putting the finishing touches on, and we define the term so I think it would be good to use classification interface here

srallen commented:
@simensta I updated the description and rebased. I think it'd be great to get #3177 in place before making this public, so I'll take a look to review your PR this week.

simensta commented:
@srallen, sounds good!

srallen commented:
#3177 is now merged, so I think all that's left is final review and then coordinating with @mrniaboc if there should be an email out to project builders about the functionality.

simensta commented:
Looks good to me!  Before merging, I think the migration that moves "pan_and_zoom: true" to the workflow configuration needs to be run again. At least one project added Pan and Zoom after [Panoptes PR#1991](https://github.com/zooniverse/Panoptes/issues/1991) was closed. 

@zwolf, could you run the migration on production and staging again?

zwolf commented:
Data update was run on both staging and production this morning.

simensta commented:
Thanks @zwolf. @mrniaboc, do you want to coordinate an announcement with the merge?
